test/plugins/test_super_check.py

```diff
@@ -45,7 +45,32 @@ def {method}(xyzzy):
         expected = Message(
             msg_id='super-method-not-called',
             node=bad_node,
-            args=(method, 'TestCase'),
+            args=(method, 'unittest.case.TestCase'),
+        )
+        with self.assertAddsMessages(expected):
+            self.walk(module)
+
+    @pytest.mark.parametrize("method", [
+        "setUpTestData",
+    ])
```

Comment:
Is this just parameterized in case more methods like this get added to the base class later?

Reply:
It's the same pattern as the code above.  I thought about doubly-parameterizing one method instead.  But it got a little complex specify the base class, and the real name of the base class, etc.